packages/expo-cli/src/commands/eas-build/utils/expoUpdates.ts

```diff
@@ -0,0 +1,514 @@
+import { AndroidConfig, ExpoConfig, getConfig, getPackageJson, IOSConfig } from '@expo/config';
+import plist from '@expo/plist';
+import { UserManager } from '@expo/xdl';
+import chalk from 'chalk';
+import figures from 'figures';
+import * as fs from 'fs-extra';
+import glob from 'glob';
+import ora from 'ora';
+import path from 'path';
+import xcode from 'xcode';
+
+import { gitAddAsync } from '../../../git';
+import log from '../../../log';
+import * as gitUtils from './git';
+
+const iOSBuildScript = '../node_modules/expo-updates/scripts/create-manifest-ios.sh';
+const androidBuildScript =
+  'apply from: "../../node_modules/expo-updates/scripts/create-manifest-android.gradle"';
```

Comment:
What if `node_modules` is not in the parent directory (`../`)? For instance, someone is using a monorepo.